src/gapc.cc

```diff
@@ -136,7 +136,11 @@ static void parse_options(int argc, char **argv, Options *rec) {
       "  3 = add data types.\n"
       "  4 = add min/max yield sizes.\n"
       "  5 = add non-terminal table dimensions.\n"
-      "(Use 'dot -Tpdf out.dot' to generate a PDF.)\nDefault file is out.dot");
+      "(Use 'dot -Tpdf out.dot' to generate a PDF.)\nDefault file is out.dot")
+    ("checkpoint",
+     "enable periodic archiving/checkpointing "
+     "of the tabulated non-terminals.\n"
+     "(creates new checkpoint every 60 minutes by default)\n");
```

Comment:
I mentioned 90 minutes in our last meeting, but actually set it to 60 minutes everywhere. My mistake. I can also make this a macro so it will basically be a variable.